pkg/operator/k8sutil/prometheus.go

```diff
@@ -16,7 +17,7 @@ func getMonitoringClient() (*monitoringclient.Clientset, error) {
 	cfg, err := clientcmd.BuildConfigFromFlags("", "")
 	client, err := monitoringclient.NewForConfig(cfg)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get monitoring client. %+v", err)
+		return nil, fmt.Errorf("failed to get monitoring client. %+", err)
```

Comment:
@galexrt Did you intend to change this to `%v`? I don't believe `%+` will work.

Reply:
Nope, but I just saw there are some files in the k8sutil that were missing the license header. I added them as well in the commit.